src/Controller/DocumentController.php

```diff
@@ -114,15 +114,21 @@ public function create(CaseEntity $case, Request $request): Response
             $documentType = $document->getType();
             /** @var UploadedFile[] $files */
             $files = $form->get('files')->getData();
+            $numberOfDocuments = count($files);
+            $count = 1;
             foreach ($files as $file) {
-                $newDocument = $this->documentUploader->createDocumentFromUploadedFile($file, $documentName, $documentType);
+                $modifiedDocumentName = 1 === $numberOfDocuments ? $documentName : $documentName.' '.$count.' af '.$numberOfDocuments;
```

Comment:
If `$files` is an array you can use

```php
foreach ($files as $index => $file) {
  $modifiedDocumentName = 1 === $numberOfDocuments ? $documentName : $documentName.' '.($index+1).' af '.$numberOfDocuments;
}
```

and get rid of the `$count` variable.

Use `sprintf` for formatting the document name and consider renaming `$modifiedDocumentName` to `$formattedDocumentName`. You could also remove `$documentName` in line 113 and rewrite to 

```php
$documentName = 1 === $numberOfDocuments ? $document->getDocumentName() : $document->getDocumentName().' '.$count.' af '.$numberOfDocuments;
```

for a better naming.